test/autoLoadAdapter.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+/**
+ * Auto load adapter for tests
+ *
+ * Copyright (c) 2016 Rain Agency.
+ * Licensed under the MIT license.
+ */
+
+const AWS = require('aws-sdk');
+const DOC = require('dynamodb-doc');
+
+class AutoLoadAdapter {
```

Comment:
if we're only using this for tests then there's no need to have it actually work with dynamodb and then mock it, let's just have a simpler plain object returning promises for get and put